provider/pact-jvm-provider-maven/README.md

```diff
@@ -394,6 +395,31 @@ For example:
 </plugin>
 ```
 
+Here is how you configure the plugin to use basic authentication for verifying pacts
+
+```xml
+<plugin>
+    <groupId>au.com.dius</groupId>
+    <artifactId>pact-jvm-provider-maven</artifactId>
+    <version>4.0.1</version>
+    <configuration>
+      <serviceProviders>
+        <serviceProvider>
+          <name>provider1</name>
+          <stateChangeUrl>http://localhost:8080/tasks/pactStateChange</stateChangeUrl>
+          <pactBroker>
+              <url>http://pactbroker:1234</url>
+              <authentication>
+                  <scheme>bearer</scheme>
+                  <username>TOKEN</username>
```

Comment:
instead of re-using the `username` attribute, it would be better to introduce a `token` attribute

Reply:
done! I re-used the `username` attribute because that's how the junit extensions appear to work based on some documentation https://docs.pactflow.io/docs/troubleshooting/ but I see that other integrations don't work this way